.circleci/config.yml

```diff
@@ -109,18 +109,23 @@ jobs:
         command: bundle -v
     - restore_cache:
         keys:
-        -  prison-visits-public-{{ checksum "Gemfile.lock" }}
+        - prison-visits-public-{{ checksum "Gemfile.lock" }}
+        - prison-visits-public-
```

Comment:
Not sure this 2nd one is needed?

Reply:
Hey Paul, this is used to restore any last partial cache that would match `- prison-visits-public-` 

https://circleci.com/docs/caching/#restoring-cache